### ISSUE TITLE
Bulk updating documents

### ISSUE THREAD

**Issue by jimlambie** [dadi/api#40]
It is currently possible to send an array of documents to the API for creation, but not to send an array of documents to be updated.

Currently to update a document the ID must be passed in the request URL and the update query as the body.

```
POST /1.0/library/notices/1234 HTTP/1.1
Host: api.example.com
content-type: application/json
Authorization: Bearer 171c8c12-6e9b-47a8-be29-0524070b0c65

{ "status": "published"  }
```

It would be useful to allow sending an array of IDs to be updated:

```
POST /1.0/library/notices/1234,5678,9012,3456 HTTP/1.1
Host: api.example.com
content-type: application/json
Authorization: Bearer 171c8c12-6e9b-47a8-be29-0524070b0c65

{ "status": "published"  }
```

**Comment by jimlambie** [comment]
Or perhaps even better (more RESTful?):

```
PUT /1.0/library/notices HTTP/1.1
Host: api.example.com
content-type: application/json
Authorization: Bearer 171c8c12-6e9b-47a8-be29-0524070b0c65

{
  ids: [1234,5678,9012,3456], 
  update: { "status": "published" }
}
```

**Comment by dark12222000** [comment]
:+1:

**Comment by jimlambie** [comment]
The wiki is maintained by the community. So if there aren't any up to date instructions, we recommend you to explore the solution yourself and hopefully contribute your findings back!

**Comment by jimlambie** [comment]
:shinto_shrine:

**Comment by mingard** [comment]
https://docs.mongodb.org/manual/reference/method/Bulk.tojson/#Bulk.tojson

**Comment by jimlambie** [comment]
@mingard please open a separate issue for the Bulk operations :cake:

**Comment by jimlambie** [comment]
Closed by 22950a2
